### ISSUE TITLE
[SampleCard-StockUpdate; Config-Windows-timeline.json]: The color contrast ratio for the text "Nastag Global  Select: MSFT" have 2.82:1 does not meet requirement of at least 4.5:1

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7387]
https://microsoft.visualstudio.com/OS/_workitems/edit/37992293
